gst/tensor_converter/tensor_converter.c

```diff
@@ -641,6 +641,8 @@ gst_tensor_converter_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
 
         /** copy timestamps */
         gst_buffer_copy_into (inbuf, buf, GST_BUFFER_COPY_METADATA, 0, -1);
+
```

Comment:
Does COPY_METADATA include Timestamp? If you want to copy timestamps, then prefer GST_BUFFER_COPY_TIMESTAMPS?